samples/net/https_client/README.rst

```diff
@@ -71,19 +71,25 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
-To build the sample with Mbed TLS and TF-M for the nRF91 Series DKs, add the following to your west build command:
+Activating sample extensions
+============================
 
-.. code-block:: none
+You can activate the following optional extensions when building the sample:
```

Comment:
```suggestion
Enable the following optional extensions for the nRF91 Series DKs during the sample build process:
```

Reply:
This line is repeated in 10+ files, so I edited to:
```
You can enable the following optional extensions during the sample build process:
```